ndmg/utils/gen_utils.py

```diff
@@ -126,15 +140,26 @@ def name_tmps(basedir, basename, extension):
 
 
 def get_braindata(brain_file):
-    """
-    Opens a brain data series for a mask, mri image, or atlas.
+    """Opens a brain data series for a mask, mri image, or atlas.
     Returns a numpy.ndarray representation of a brain.
-    **Positional Arguements**
-        brain_file:
-            - an object to open the data for a brain.
-            Can be a string (path to a brain file),
-            nibabel.nifti1.nifti1image, or a numpy.ndarray
+    
+    Parameters
+    ----------
```

Comment:
`brain_file : object` should be `brain_file : str, nibabel.nifti1.nifti1image, numpy.ndarray`